### PR TITLE
clean up php error log

### PR DESCRIPTION
**Pull request by gymad**
## Description

This PR fix some PHP notice/warning etc. in php error log and shows the possible problems in suitecrm log
## Motivation and Context

errors in the php error.log
## How To Test This

This changes don't change the code behavior
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Import/views/view.error.php [hunk 1]
```diff
@@ -75,25 +81,31 @@ public function getMenu(
  	protected function _getModuleTab()
     {
         global $app_list_strings, $moduleTabMap;
-        
+
  		// Need to figure out what tab this module belongs to, most modules have their own tabs, but there are exceptions.
         if ( !empty($_REQUEST['module_tab']) )
             return $_REQUEST['module_tab'];
-        elseif ( isset($moduleTabMap[$_REQUEST['import_module']]) )
+        elseif ( isset($_REQUEST['import_module']) && $_REQUEST['impoert_module'] && isset($moduleTabMap[$_REQUEST['import_module']]) )
```
Comment (samus-aran) [inline modules/Import/views/view.error.php]: import typo

File: modules/Import/views/view.error.php [hunk 2]
```diff
@@ -75,25 +81,31 @@ public function getMenu(
...
+        }
  	}
  	
  	/** 
      * @see SugarView::display()
      */
  	public function display()
     {
-        $this->ss->assign("IMPORT_MODULE", $_REQUEST['import_module']);
+        $this->ss->assign("IMPORT_MODULE", isset($_REQUEST['import_module']) && $_REQUEST['impoert_module'] ? $_REQUEST['import_module'] : null);
```
Comment (samus-aran) [inline modules/Import/views/view.error.php]: import typos still

File: modules/Import/views/view.error.php [hunk 3]
```diff
@@ -75,25 +81,31 @@ public function getMenu(
...
+
  		// Need to figure out what tab this module belongs to, most modules have their own tabs, but there are exceptions.
         if ( !empty($_REQUEST['module_tab']) )
             return $_REQUEST['module_tab'];
-        elseif ( isset($moduleTabMap[$_REQUEST['import_module']]) )
+        elseif ( isset($_REQUEST['import_module']) && $_REQUEST['impoert_module'] && isset($moduleTabMap[$_REQUEST['import_module']]) )
             return $moduleTabMap[$_REQUEST['import_module']];
         // Default anonymous pages to be under Home
-        elseif ( !isset($app_list_strings['moduleList'][$_REQUEST['import_module']]) )
+        elseif ( isset($_REQUEST['import_module']) && $_REQUEST['impoert_module'] && !isset($app_list_strings['moduleList'][$_REQUEST['import_module']]) )
```
Comment (samus-aran) [inline modules/Import/views/view.error.php]: import typos

File: export.php [hunk 4]
```diff
@@ -77,7 +77,7 @@
 
 $transContent = $GLOBALS['locale']->translateCharset($content, 'UTF-8', $GLOBALS['locale']->getExportCharset());
 
-if($_REQUEST['members'] == true)
+if(isset($_REQUEST['members']) && $_REQUEST['members'] == true)
```
Comment (daniel-samson) [inline export.php]: Use {} for if / while.

File: modules/Import/views/view.error.php (hunk 1 above)
Comment (daniel-samson) [inline modules/Import/views/view.error.php]: please use {} if / while statements

File: modules/Import/views/view.error.php [hunk 5]
```diff
@@ -75,25 +81,31 @@ public function getMenu(
  	protected function _getModuleTab()
     {
         global $app_list_strings, $moduleTabMap;
-        
+
  		// Need to figure out what tab this module belongs to, most modules have their own tabs, but there are exceptions.
         if ( !empty($_REQUEST['module_tab']) )
             return $_REQUEST['module_tab'];
-        elseif ( isset($moduleTabMap[$_REQUEST['import_module']]) )
+        elseif ( isset($_REQUEST['import_module']) && $_REQUEST['import_module'] && isset($moduleTabMap[$_REQUEST['import_module']]) )
```
Comment (daniel-samson) [inline modules/Import/views/view.error.php]: Coding standards: Please add {} to if else.

File: modules/ModuleBuilder/views/view.property.php [hunk 6]
```diff
@@ -77,7 +77,10 @@ protected function _getModuleTitleParams($browserTitle = false)
     }
 
 
-    function init () // pseduo-constuctor - given a well-known name to allow subclasses to call this classes constructor
+    function init (
+        $bean = null,
+        $view_object_map = array()
+    ) // pseduo-constuctor - given a well-known name to allow subclasses to call this classes constructor
```
Comment (daniel-samson) [inline modules/ModuleBuilder/views/view.property.php]: This comment should be in the PHPDoc section above the method.

Comment (gymad) [inline modules/ModuleBuilder/views/view.property.php]: added to PHPDoc section

**Comment by Mac-Rae** [comment]
Please resolve the conflicts on this pull request bringing it up to date.

**Pull request closed without merging**